applications/workers/release/combined-worker/build.gradle

```
@@ -20,11 +20,28 @@ quasar {
     ])
 }
 
+configurations {
+    drivers {
+        canBeConsumed = false
+        transitive = false
+    }
+}
+
 dependencies {
     // for BundleManager
     compileOnly "org.osgi:osgi.core"
     compileOnly 'org.osgi:org.osgi.service.component.annotations'
 
+    drivers "org.postgresql:postgresql:$postgresDriverVersion"
+
+    // Oracle isn't OSGi wrapped, but if it were, it is sufficient to just do this:
+    // OSGi bundle that registers a `DataSourceFactory` on behalf of Oracle,
+    // and add the OSGi wrapped version, i.e.
+    //
+    // drivers "org.ops4j.pax.jdbc:pax-jdbc-oracle:1.5.3"
+    // drivers "com.oracle.database.jdbc:ojdbc10WRAPPED-AS-A-BUNDLE:19.15.0.0"
```

@barrylapthorn , I tried to reword this a little bit to make it less specific, but I'm not sure if it is still correct?

@mnesbit , are you ok for us to mention Oracle here in the comments? It doesn't have an Oracle reference, or promises Oracle support, but it's helpful as an example.

```suggestion
    // JDBC drivers need to have the OSGi metadata and use `DataSourceFactory`.
    // Some drivers don't have this (e.g. Oracle) and need to be "wrapped", once they are, we can do this:
    // Specify OSGi bundle that registers a `DataSourceFactory` on behalf of <db-driver> (e.g. Oracle),
    // and add the OSGi wrapped version, i.e.
    //
    // drivers "org.ops4j.pax.jdbc:pax-jdbc-<db-driver>:1.5.3"
    // drivers "<db-driver>.jdbc:ojdbc10WRAPPED-AS-A-BUNDLE:19.15.0.0"
```

---

Arguably, the same information is capture on the wiki page:  https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4105502757/JDBC+and+OSGi

We could remove that from the comments altogether - though an example *is* in the comments because a developer is more likely to see these comments.

---

I agree it's useful to be here.

---

Updated with "vendor" instead.